opentoall-ctf-2015/misc/secure-server/README.md

```diff
@@ -42,7 +42,7 @@ We see a `robots.txt` file that gives us another directory to search for any hid
 Inside `http://104.131.124.226:8080/secret`, we find a [secret.zip](./104.131.124.226\:8080/secret/secret.zip) file that we download and extract:
 
 ```bash
-$ unzip secret.zip && tree -a
+$ unzip secret.zip \&\& tree -a
```

Comment:
I did revert that one. @initbar should mention the editor used, though, which breaks on a `'`, `&` or `"`. Failed to verify before, thought it is a similar problem as `_` in vim for another editor.